doc/cascadia/profiles.schema.json

```diff
@@ -64,6 +64,8 @@
         "renameTab",
         "commandPalette",
         "wt",
+        "closeOtherTabs",
```

Comment:
I've usually seen this phrased as "Close all but this" and the other one as "close all to the right" paired with another option that's "close all to the left" in things like Visual Studio and Notepad++

Reply:
Somewhere, sometime, someone mentioned that we should avoid using "left"/"right" when we ideologically mean "next"/"previous". Something about how for RTL UI's the "next" action is actually to the left, or when someone adds support for a vertical arrangement of tabs, or something like that. It really stuck with me so I've been thinking about it ever since. We could always change the generated name to say "Close all but the tab at index {0}" and "Close all tabs to the right of tab {0}", I don't really care.

Reply:
I didn't think of the left/right being mirrored for RTL. That's a good call for the relative positioning ones. 

I do still sort of prefer "close all but this" as phrasing, but I'm not going to push it that hard.